CIP-0095/README.md

```diff
@@ -0,0 +1,1005 @@
+the Conway Ledger Era and thus lacks the required methods to support full user
+interaction. We believe that expecting existing CIP-30 implementors to upgrade
+implementations is unfeasible, thus we must extend it's functionality with this
+API.
+
+This proposal enables Ada holders, and DReps to engage web-based tooling through
+wallets. Thus the primary stakeholders for this proposal are tool developers and
+wallet providers. Here we aim to outline all endpoints needed to be exposed to
+web based tools to support all the needs Ada holders and DReps to engage with
+CIP-1694's governance design.
+
+## Specification
+
+We define the following section as an extension to the specification described
+within CIP-30.
+
+> **Note** This specification will evolve as the proposed ledger governance
+> model matures.
+
+### DRep Key
+
+// TODO: Move this into a more appropriate **separate** CIP.
+
+The Conway ledger era introduces a new _first class_ credential in
+[`voting_credential`](https://github.com/input-output-hk/cardano-ledger/blob/master/eras/conway/test-suite/cddl-files/conway.cddl#L329).
+This is used to identify registered DReps on-chain, via their certificates and
+votes.
+
+CIP-1694 does not define a derivation path for these registered DRep
+credentials, here we propose the introduction of DRep Keys to be used as DRep
+`voting_credential`s for (non-script) registered DReps.
+
+The methods described here should not be considered the definitive method of
+generating DRep credentials. Rather these methods should be employed by wallets
+to derive keys for non-script registered DReps.
+
+#### Derivation
+
+Here we describe DRep Key derivation as it attains to Cardano wallets who follow
+the
+[CIP-1852 | HD (Hierarchy for Deterministic) Wallets for Cardano](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/README.md)
+standard.
+
+To differentiate DRep credentials from other Cardano keys the derivation path
+must follow:
+
+`m / 1694' / 1815' / account' / 1718 / address_index`
+
+> **Note** `1718` was the year that François-Marie adopted the pseudonym
+> Voltaire.
+
+We strongly suggest that a maximum of one set of DRep credentials should be
+associated with one wallet account, this can be achieved by only ever setting
+`address_index=0`. This avoids the need for DRep Key discovery.
+
+We believe the overhead that would be introduced by "multi-DRep" accounts is an
+unjustified expense. Future iterations of this specification may expand on this,
+but at present this is seen as unnecessary.
+
+#### Tooling
+
+Supporting tooling should clearly label these key pairs as "CIP-95 DRep Keys".
+
+Bech32 prefixes of `drep_sk` and `drep_vk` should be used, as described in
+[CIP-0005](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0005/README.md).
+
+Examples of acceptable `keyType`s for supporting tools:
+
+| `keyType`                          | Description           |
```

Comment:
Lets keep this simple and mirror what we already have for other derived keys, e.g.:
`DRepExtendedSigningKey_ed25519_bip32`

The descriptions can be whatever, but we shouldn't distinguish in the types for HW vs non-hw in the vkeys (and hw signing would never be saived to disk anyways)

Reply:
Check [fbc5fcb](https://github.com/cardano-foundation/CIPs/pull/509/commits/fbc5fcbb127313ccfd2a30376145f63627f3afd9) -- this should amend.